superset/dashboards/api.py

```diff
@@ -41,8 +43,11 @@
     DashboardPutSchema,
     get_delete_ids_schema,
     get_export_ids_schema,
+    thumbnail_query_schema,
 )
 from superset.models.dashboard import Dashboard
+from superset.tasks.thumbnails import cache_dashboard_thumbnail
+from superset.utils.selenium import DashboardScreenshot
```

Comment:
Same comment here - could we move DashboardScreenshot to a module that isn't named after the specific schema?